enterprise/backend/src/metabase_enterprise/serialization/v2/ingest/yaml.clj

```diff
@@ -0,0 +1,44 @@
+(ns metabase-enterprise.serialization.v2.ingest.yaml
+  (:require [clojure.java.io :as io]
+            [metabase-enterprise.serialization.v2.ingest :as ingest]
+            [yaml.core :as yaml])
+  (:import java.io.File))
+
+(defmulti ^:private build-metas
+  (fn [^File file] (.getName file)))
+
+(defmethod build-metas "settings.yaml" [file]
```

Comment:
Should we key on this using the model type?

just like how we're doing on line 36 to make it consistence?